src/test/java/core/basesyntax/ReaderServiceTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final String INPUT_TEST_PATH = "src/test/java/core/basesyntax"
+            + "/recources/inputFileTest1.csv";
+    private static final String INCORRECT_TEST_PATH = "src/test/java/core/basesyntax"
+            + "/recources/input.csv";
+    private List<String> expected;
+    private ReaderService readerService;
+
+    @Before
+    public void setUp() {
+        readerService = new ReaderServiceImpl();
```

Comment:
Maybe initialization of readerService could be in beforeClass method, because we can use the same instance in each of our tests